### PR TITLE
release 3.0.0

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,9 +2,17 @@
...
 
-## [v2.4.0](https://github.com/voxpupuli/metadata-json-lint/tree/v2.4.0) (2020-06-12)
+## [v2.5.0](https://github.com/voxpupuli/metadata-json-lint/tree/v2.5.0) (2020-11-17)
 
-[Full Changelog](https://github.com/voxpupuli/metadata-json-lint/compare/2.3.0...v2.4.0)
+[Full Changelog](https://github.com/voxpupuli/metadata-json-lint/compare/2.4.0...v2.5.0)
+
+**Merged pull requests:**
+
+- Require Ruby 2.1 and drop post\_install\_message [\#112](https://github.com/voxpupuli/metadata-json-lint/pull/112) ([ekohl](https://github.com/ekohl))
```
Comment (ekohl) [inline CHANGELOG.md:11]: Is this a minor release? Not sure how much pins this module and how painful a major release is.

Comment (bastelfreak) [reply CHANGELOG.md:11]: we could also do a major release, I just wasn't 100% sure about it.

Comment (ekohl) [reply CHANGELOG.md:11]: I'm leaning to a major release since it's dropping an older Ruby version.

Comment (bastelfreak) [reply CHANGELOG.md:11]: updated it

File: README.md [hunk 2]
```diff
@@ -81,3 +81,60 @@ export CHANGELOG_GITHUB_TOKEN=...
 ## Contributors
 
 A big thank you to the [contributors](https://github.com/voxpupuli/metadata-json-lint/graphs/contributors).
+
+## Making a new release
```
Comment (ekohl) [inline README.md:85]: Right above `Contributors there's a chapter `Make a new release`. Please merge these 2.

**Review by ekohl — CHANGES_REQUESTED** [review]
:+1: on the 3.0.0 release but a comment about the docs.

**Review by ekohl — APPROVED** [review]

**Pull request merged**